app/controllers/user_invite_requests_controller.rb

```diff
@@ -21,7 +21,7 @@ def new
         redirect_to login_path
      end
     else
-      flash[:error] = ts("Sorry, new invitations are temporarily unavailable. If you are the mod of a challenge currently being run on the Archive, please <a href=\"#{new_feedback_report_url}\">contact Support</a>. If you are the maintainer of an at-risk archive, please contact <a href=\"http://opendoors.transformativeworks.org/contact/open doors\">Open Doors</a>".html_safe)
+      flash[:error] = ts("Sorry, additional invitations are unavailable. Please <a href=\"http://archiveofourown.org/invite_requests\">use the queue</a>! If you are the mod of a challenge currently being run on the Archive, please <a href=\"#{new_feedback_report_url}\">contact Support</a>. If you are the maintainer of an at-risk archive, please contact <a href=\"http://opendoors.transformativeworks.org/contact-open-doors/\">Open Doors</a>".html_safe)
```

Comment:
I think we want to do this as #{invite_requests_path} rather than hard code it in as the full URL.